selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithRootFolderTest.java

```diff
@@ -51,14 +46,18 @@
   @Inject private SeleniumWebDriver seleniumWebDriver;
   @Inject private TestWorkspaceServiceClient workspaceServiceClient;
   @Inject private TestProjectServiceClient projectServiceClient;
-  @Inject private Dashboard dashboard;
+  @Inject private TestGitHubRepository testRepo;
 
   private TestFactory testFactoryWithRootFolder;
 
   @BeforeClass
   public void setUp() throws Exception {
-    testFactoryWithRootFolder =
-        testFactoryInitializer.fromUrl("https://github.com/" + gitHubUsername + "/quickstart");
+    // preconditions - create test the repository
```

Comment:
Test repo has actually created earlier, and here we are adding project into the test repo.